### PR TITLE
created news and courses page

### PR DESCRIPTION
**Pull request by willyserge**
###  what does this PR do?

- created courses page
- re-designed news page

### NOTE: only review needed on this. do not merge into develop

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -1,5 +1,69 @@
...
+         totalCount
+       }
+     }
+  `);
+
+  if (errors) {
+    reporter.panicOnBuild('Error while running a GraphQL query!');
+    return;
+  }
+
```
Comment (eliemugenzi) [inline gatsby-node.js:17]: Revert these changes since they are in another PR to avoid conflicts.

File: gatsby-node.js [hunk 2]
```diff
@@ -13,30 +77,17 @@ exports.onCreateNode = ({node, getNode, actions}) => {
...
-        allMarkdownRemark{
-          nodes {
-            fields {
-              slug
-            }
-          }
-        }
-      }
-      `)
-      .then((result) => {
```
Comment (eliemugenzi) [inline gatsby-node.js:29]: Get this back to how it was.

File: gatsby-node.js (hunk 1 above)
Comment (willyserge) [reply gatsby-node.js:17]: ok.its being handled
